docs/upgrading-fcos.md

```diff
@@ -15,11 +15,70 @@ Occasionally, changes are made to Fedora CoreOS Butane configs (those that speci
 
 {% comment %}
 
+## From Version 1.5.0 to Version 1.6.0-experimental
+
+There are no breaking changes between versions 1.5.0 and 1.6.0-experimental of the `fcos` configuration specification. Any valid 1.5.0 configuration can be updated to a 1.6.0 configuration by changing the version string in the config.
+
 ## From Version 1.4.0 to Version 1.5.0
 
 There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` configuration specification. Any valid 1.4.0 configuration can be updated to a 1.5.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
+### Support Luks discard
+
+Discard; similarly to trim, improves performance for SSDs.
```

Comment:
Please try to mimic the format of other entries in this file.  In many cases you can just reuse the text from the Ignition upgrade notes.

Reply:
Ah, yeah I missed the documentation section of the upgrade docs checklist. I see that now. Thank you